net/aircrack-ng/patches/0001-osdep-ratiotap-fix-compilation-on-Linux.patch

```diff
@@ -0,0 +1,36 @@
+From 85fde8b32b2095d60cead683c958487701221125 Mon Sep 17 00:00:00 2001
+From: Yousong Zhou <yszhou4tech@gmail.com>
+Date: Wed, 30 Dec 2015 10:39:56 +0800
+Subject: [PATCH] osdep: ratiotap: fix compilation on Linux
+
+le16toh and le32toh should be already available by platform.h
```

Comment:
I compile and run-tested this against QEMU big endian malta.  Without this patch the following compilation error popped out.  It's not yet sent upstream because I don't have much experience with BSD dev environment.

```
osdep/radiotap/radiotap.o: In function `ieee80211_radiotap_iterator_init':
/home/yousong/git-repo/trunk-openwrt/build_dir/target-mips_24kc_musl-1.1.11/aircrack-ng-1.2-rc3/src/osdep/radiotap/radiotap.c:123: undefined reference to `bswap_32'
/home/yousong/git-repo/trunk-openwrt/build_dir/target-mips_24kc_musl-1.1.11/aircrack-ng-1.2-rc3/src/osdep/radiotap/radiotap.c:143: undefined reference to `bswap_32'
osdep/radiotap/radiotap.o: In function `ieee80211_radiotap_iterator_next':
/home/yousong/git-repo/trunk-openwrt/build_dir/target-mips_24kc_musl-1.1.11/aircrack-ng-1.2-rc3/src/osdep/radiotap/radiotap.c:386: undefined reference to `bswap_32'
```